### DISCUSSION TITLE
how to launch Zettlr on debian (Linux) install from terminal

### DISCUSSION THREAD

**Discussion by cedardocster** [Zettlr/Zettlr#4108] [General]
Public service announcement:  

on linux, to launch Zettlr from the terminal you have to capitalize Zettlr for it to work (zettlr = no joy)

I didn't try it until I was writing this post as most linux applications started at the terminal are lower case.

(Nothing more to see here ;)

**Comment by nathanlesage** [comment]
The upper case is actually due to some weirdness in our build pipeline -- if we lowercase the binary on Unixoid systems, the app stops working on macOS or something 🙄 But it's even documented, that the binary on Linux is oddly capitalised.
